logging/Terraform.md

```diff
@@ -0,0 +1,62 @@
+## Requirements
```

Comment:
To keep it consistent with other modules:

1. Rename this file to terraformoptions.adoc
2. Use asciidoc to document

Reply:
Will make the change.